traffic_ops/etc/profile.d/traffic_ops.sh

```diff
@@ -12,11 +12,4 @@
 # limitations under the License.
 #
 
-TO_DIR="/opt/traffic_ops/app"; export TO_DIR
-
-# Setup GOPATH
-GOPATH="/opt/traffic_ops/go"; export GOPATH
-GOBIN=$GOPATH/bin
-
-# Setup PATH
-PATH=$PATH:$GOBIN:/usr/local/go/bin
+export TO_DIR=/opt/traffic_ops/app;
```

Comment:
IMO the benefit is being able to look at a string definition at a glance and know that, because there are no quotes but ShellCheck linting passes, it contains no spaces.

On a side note, ShellCheck would not complain if there was an unquoted variable reference like `$TO_DIR` later in this file, instead of `"$TO_DIR"`, because it notes  that `$TO_DIR` contains no spaces. Quoting variable references always makes sense to me, though...